samcli/lib/cookiecutter/interactive_flow_creator.py

```diff
@@ -42,6 +42,14 @@ def create_flow(flow_definition_path: str, extra_context: Optional[Dict] = None)
                         "True": "key of the question to jump to if the user answered 'Yes'",
                         "False": "key of the question to jump to if the user answered 'Yes'",
                       }
+                      "default": "default_answer",
+                      "default_from_toml": {
+                        "toml_file": "path/to/toml/file.toml",
+                        "env": {"valueof": "key-of-another-question-to-resolve-value-from-its-answer"},
+                        "cmd_names": "command_names",
+                        "section": "parameters",
+                        "key": "pipeline_user",
+                      }
```

Comment:
From a template writer UX perspective, I feel quite hard to understand this. Do we expect the template writer to know "cmd_names" and "env?"

can we remove the "cmd_names" part because we can provide this value in pipeline command. 

And for "env," it is a concept that seems not introduced in this context, how do we expect them to figure out what `env` and `valueof` mean?